tests/console/yast2_samba.pm

```diff
@@ -216,7 +216,7 @@ sub run {
     wait_serial('yast2-samba-server-status-0', 60) || die "'yast2 samba-server' didn't finish";
 
     # check samba server status
-    assert_script_run("systemctl show -p ActiveState smb.service | grep ActiveState=active");
+    systemctl 'show -p ActiveState smb.service | grep ActiveState=active';
```

Comment:
As above, not only action on service. Maybe worth adding functionality to systemctl subroutine.